web/docs/faqs.md

```diff
@@ -4,43 +4,51 @@ This page provides answers to frequently asked questions (FAQs) about Tenzir.
 
 ## What is Tenzir?
 
-Tenzir is a platform that pioneers open-source security data pipelines.
+Tenzir is the data pipeline engine for security teams, enabling you to
+collect, parse, shape, normalize, aggregate, route, store, and query your
+security telemetry at ease.
 
-Tenzir is also the name of the Germany-based startup behind the product.
+Tenzir is also the name of the startup behind the product.
 
 ## What part of Tenzir is open and what part is closed source?
 
-The diagram below illustrates the high-level components, indicating what parts
-are *open* and what are *closed* source:
+To create a healthy open core business with a thriving open source foundation,
+we aim to find the right balance between enabling open source enthusiast whilst
+offering a commercial package for those seeking a turn-key solution:
 
-![Open source vs. closed source](open-vs-closed-source.excalidraw.svg)
+![Open source vs. closed source](open-vs-closed-source.svg)
 
-The majority of code is open source and available at under a permissive BSD
-3-clause licence, available at [GitHub](https://github.com/tenzir/tenzir). This
-code implements the `tenzir` and `tenzir-node` [command line
-tools](configuration.md) that execute pipelines and store data. A flexible
-[plugin infrastructure](architecture/plugins.md) allows for enhancing the
-open-source core with closed-source plugins.
+There exist three moving parts:
 
-The Tenzir stack also consists of a cloud platform and web app, accessible via
-[app.tenzir.com](https://app.tenzir.com). Both of these components are not
-openly available. The platform is the *control plane* that manages the fleet of
-nodes and their pipelines, whereas the nodes are the *data plane* for compute
-and storage.
+1. **Executor**: The pipeline executor is open source and available at under a
+   permissive BSD 3-clause licence at
+   [GitHub](https://github.com/tenzir/tenzir). The `tenzir`
+   command line tool is the primary interface.
+2. **Node**: The node makes pipeline management easy. A node orchestrates
+   multiple pipelines and offers additional features, such as contexts for
+   enrichment and an indexed storage engine.
```

Comment:
Should this mention the `tenzir-node` binary, similar to how the first bullet mentions the `tenzir` binary?

Reply:
I will remove the `tenzir` binary part instead. This is not that relevant at this flying altitude.